grantnav/frontend/views.py

```diff
@@ -717,7 +717,14 @@ def search(request, template_name="search.html"):
         context['results'] = results
         context['json_query'] = json.dumps(json_query)
         context['query'] = json_query
-        context['view_mode'] = request.GET.get('view_mode')
+        view_mode = request.GET.get('view_mode')
+        if view_mode:
```

Comment:
Not a blocker - could probably use the new := operator here? I do wonder if this could be simplified too? No problem if it can't be.

Reply:
Update to the walrus operator.